docs/src/components/BadgeWithParentExample/BadgeWithParentExample.component.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+    This component exists for the badge.mdx documentation.
+*/
+
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled, { ThemeProvider } from '@xstyled/styled-components';
+// COMPONENTS
+import { Badge, Cart } from '@retailmenot/anchor';
+// THEME
+import { colors, RootTheme } from '@retailmenot/anchor';
+
+const { useState } = React;
+
+const StyledBadgeWithParentExample = styled.div`
+    padding: 2rem 5rem;
+    background-color: ${colors.grapePurchase.base};
+    color: ${colors.white.base};
+    line-height: 1.5rem;
+
+    &:hover {
+        background-color: ${colors.grapePurchase.light};
+    }
+`;
+
+export const BadgeWithParentExample = () => {
+    const [isHovered, setIsHovered] = useState<boolean>(false);
+    const handleOnMouseEnter = () => {
+        setIsHovered(true);
+    };
+    const handleOnMouseLeave = () => {
+        setIsHovered(false);
+    };
+
+    return (
+        <ThemeProvider theme={RootTheme}>
```

Comment:
You don't need to import `ThemeProvider` or `RootTheme`, they are included at the top of the app.

Reply:
✅ Will update